### PR TITLE
added cfw-calender

### PR DESCRIPTION
**Pull request by nullx002**
support for cfw-calendar added

### CODE REVIEW

**Comment by wasamasa** [comment]
One big thing sticking out: You haven't stuck the theme block into the correct position. C doesn't come after W...

**Comment by nullx002** [comment]
tried few colours for the foreground but none worked like yellow, also removing it altogether is not helping with the highlight. moved code to C, didn't knew it was ordered alphabetically so added it just below previous code that i added. also merge and rebase works slightly differently in my version of magit than you explained but finally i figured it out after a bit of experimentation. also learned a few more things about git in the process. thanks for help.

File: gotham-theme.el [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 ;; Author: Vasilij Schneidermann <v.schneidermann@gmail.com>
 ;; URL: https://github.com/wasamasa/gotham-theme
+;; Package-Version: 20170414.1046
```
Comment (wasamasa) [inline gotham-theme.el]: You shouldn't touch this either.

**Comment by nullx002** [comment]
sorry, done.

**Comment by wasamasa** [comment]
OK, one more thing: The commit message doesn't reflect your changes, please reword it to something like "Add calfw faces".

**Comment by nullx002** [comment]
ok, done

**Comment by wasamasa** [comment]
Great, thanks!
